Lib/importlib/machinery.py

```diff
@@ -27,3 +27,13 @@ def all_suffixes():
            'NamespaceLoader', 'OPTIMIZED_BYTECODE_SUFFIXES', 'PathFinder',
            'SOURCE_SUFFIXES', 'SourceFileLoader', 'SourcelessFileLoader',
            'WindowsRegistryFinder', 'all_suffixes']
+
+
+def __getattr__(name):
+    if name in ('DEBUG_BYTECODE_SUFFIXES','OPTIMIZED_BYTECODE_SUFFIXES', 'WindowsRegistryFinder' ):
+        warnings.warn(f"The '{name}' module suffix is deprecated.",
```

Comment:
The message mentions a suffix but `WindowsRegistryFinder` is not a suffix. Maybe the message could be improved?